app/main/views/index.py

```diff
@@ -45,6 +45,8 @@ def index():
         scrollTo="false",
         admin_base_url=current_app.config['ADMIN_BASE_URL'],
         stats=get_latest_stats(get_current_locale(current_app)),
+        csv_max_rows=current_app.config['CSV_MAX_ROWS'],
+        default_free_sms_fragment_limits_central=current_app.config.get('DEFAULT_FREE_SMS_FRAGMENT_LIMITS', {}).get('central', 10000),
```

Comment:
```suggestion
        default_free_sms_fragment_limits_central=current_app.config['DEFAULT_FREE_SMS_FRAGMENT_LIMITS']['central'],
```
Don't want to fallback to another value if this config dict gets updated, we would need to change code here and this is expected